src/Data/YAML/Token.hs

```diff
@@ -551,7 +551,7 @@ parser ?! decision = peek parser *> commit decision
 (>?) :: (Match match result) => match -> Parser result
 (>?) lookahead = peek lookahead
 
--- | @lookahead >?@ matches the current point without consuming any characters
+-- | @lookahead >! matches the current point without consuming any characters
```

Comment:
Thanks for spotting.
This introduced another typo, though: missing closing markup `@`.